packages/backend/src/machine-utils.ts

```diff
@@ -97,14 +97,17 @@ export async function isPodmanV5Machine() {
 }
 
 export async function checkPrereqs(): Promise<string | undefined> {
-  const isPodmanV5 = await isPodmanV5Machine();
-  if (!isPodmanV5) {
-    return 'Podman v5.0 or higher is required to build disk images.';
-  }
+  // Podman Machine checks are applicable to non-Linux platforms only
+  if (!isLinux()) {
```

Comment:
should still have this check here since we use checkPrereqs() from the frontend.

the reason why this is in the backend rather than frontend is that we do too much loading in onMount() in build.svelte, specifically with the update() function which causes issues trying to pre-detect Linux.

So it's better to check when executing the backend commands instead if we are or not on Linux

Reply:
I meant even on Linux shouldn't we confirm podman is installed? (vs this code which has no prereq checking for linux)

Reply:
Ohhh! There's no check anyways because the Build page will already error out with no Podman installed / never reach this point.

We should actually add this to the dashboard / have this higher up?

I opened up: https://github.com/containers/podman-desktop-extension-bootc/issues/701